scripts/partition_manager_files.py

```diff
@@ -0,0 +1,89 @@
+
+import argparse
+import yaml
+from os import path
+
+
```

Comment:
IMO the `partition_manager_files`  name is a bit to ambiguous, is it input files or output files?

Could we call it `partition_manager_output` or something like that?

Reply:
`export`, `representation`

Reply:
`partition_mananger_file_emitter`?

Reply:
I went with `partition_manager_output.py`.